build/Makefile

```
@@ -57,7 +57,7 @@ KIND_PROFILE ?= agones
 KIND_CONTAINER_NAME=$(KIND_PROFILE)-control-plane
 
 # Game Server image to use while doing end-to-end tests
-GS_TEST_IMAGE ?= gcr.io/agones-images/udp-server:0.21
+GS_TEST_IMAGE ?= gcr.io/agones-images/game-server:0.1
```

```suggestion
GS_TEST_IMAGE ?= gcr.io/agones-images/simple-game-server:0.1
```

Just saw this after I kicked off the build :man_facepalming: sorry I missed it.

---

Ah, how could I miss that one. Thanks